alicloud/resource_alicloud_ess_alarm.go

```diff
@@ -142,6 +147,19 @@ func resourceAliyunEssAlarmCreate(d *schema.ResourceData, meta interface{}) erro
 	}
 	response, _ := raw.(*ess.CreateAlarmResponse)
 	d.SetId(response.AlarmTaskId)
+	// enable or disable alarm
+	enable := d.Get("enable")
+	if !enable.(bool) {
+		disableAlarmRequest := ess.CreateDisableAlarmRequest()
+		disableAlarmRequest.AlarmTaskId = response.AlarmTaskId
+		raw, err = client.WithEssClient(func(essClient *ess.Client) (interface{}, error) {
+			return essClient.DisableAlarm(disableAlarmRequest)
+		})
+		if err != nil {
+			return WrapErrorf(err, DefaultErrorMsg, "alicloud_ess_alarm", disableAlarmRequest.GetActionName(), AlibabaCloudSdkGoERROR)
```

Comment:
I think the line 159 should be `			return WrapErrorf(err, DefaultErrorMsg, d.Id(), disableAlarmRequest.GetActionName(), AlibabaCloudSdkGoERROR)
`

Reply:
fixed